app/views/replies/_write.haml

```diff
@@ -11,8 +12,9 @@
     = f.hidden_field :character_id
     = f.hidden_field :icon_id
     = f.hidden_field :character_alias_id
+    = f.hidden_field :editor_mode, value: editor
     = hidden_field_tag :allow_dupe, @allow_dupe
-    = hidden_field_tag :editor_mode, params[:editor_mode] || current_user.default_editor
+    = hidden_field_tag :editor_mode, editor
```

Comment:
hey past me why does that have both a hidden field tag and a form hidden_field

Reply:
A good question - I guess we can remove this second one?